catalog/spatial/wfs/2.0.0/spatial-wfs-v2_0_0-source/src/main/java/org/codice/ddf/spatial/ogc/wfs/v2_0_0/catalog/source/WfsSource.java

```diff
@@ -518,8 +519,21 @@ private void buildFeatureFilters(List<FeatureTypeType> featureTypes,
 
             LOGGER.debug("ftName: {}", ftSimpleName);
             try {
-                XmlSchema schema = wfs.describeFeatureType(new DescribeFeatureTypeRequest(
-                        featureTypeType.getName()));
+                XmlSchema schema =
+                        wfs.describeFeatureType(new DescribeFeatureTypeRequest(featureTypeType.getName()));
+
+                if (schema == null) {
```

Comment:
We can use the `@Encoded` annotation on the Wfs CXF method in question. Not sure how this will affect security though, and this was a special case we ran into with one server. @stustison thoughts?